README.md

```diff
@@ -10,24 +10,40 @@
 # cromshell
  A script for submitting workflows to a cromwell server and monitoring / querying their results.
 
-requires `column`, `curl`, `mail`, and [jq](https://stedolan.github.io/jq/)
+Current version: 2.0.0.beta 
 
-Current version: 2.0.0 
+Cromshell 2 is the next step in the evolution Cromshell. It offers many of the same
+functions as Cromshell 1 but has been rebuilt in python with many added benefits such as:
+- Added integration and unit tests to insure reliability 
+- Multiple way of installation (source, brew tap, and pypi)
+- Moduler code architecture that makes easy to understand so users are encouraged 
+to submit features they'd like to see in the tool.
+- Developer documentation showing how to add new features and commands to the tool.
 
 ## Examples:
 
 ```
-         cromshell-alpha submit workflow.wdl inputs.json options.json dependencies.zip
-         cromshell-alpha status
-         cromshell-alpha -t 20 metadata
-         cromshell-alpha logs -2
+         cromshell-beta submit workflow.wdl inputs.json options.json dependencies.zip
+         cromshell-beta status
+         cromshell-beta -t 20 metadata
+         cromshell-beta logs -2
 ```
 
 ## Supported Flags:
+  * `--no_turtle` or `--I_hate_turtles`
+    * Hide turtle logo
+  * `--cromwell_url` `TEXT`
```

Comment:
if it takes an argument, technically it's not a flag

Reply:
Changing to "Supported Options"